e2e/__tests__/__snapshots__/globals.test.js.snap

```diff
@@ -37,7 +37,7 @@ exports[`cannot have describe with no implementation 1`] = `
 
 exports[`cannot have describe with no implementation 2`] = `
 "Test Suites: 1 failed, 1 total
-Tests:       0 total
+Tests:       1 total
```

Comment:
this is wrong, there are no executed tests

Reply:
Ah yes I think I know what this is. Good catch.

Jest has some logic for telling you when a test suite fails to run (like a syntax error or something) and I think my logic for flagging that as an error needs adjusting.